tests/drivers/spi/spi_loopback/src/spi.c

```diff
@@ -149,12 +149,20 @@ static int spi_complete_loop(struct spi_config *spi_conf)
 			.len = BUF_SIZE,
 		},
 	};
+	struct spi_buf_array tx = {
```

Comment:
All the structures added in this file could be `const`. `rx_bufs` as well now.

Reply:
seems like only tx and rx can be const, not tx_bufs nor rx_bufs, as they go to bufs into spi_buf_array, which is not const (or then we have the spi_context.h issue)